modules/hcp-aws-tags.adoc

```diff
@@ -0,0 +1,328 @@
+----
+
+. Delete the operator pod or roll out a restart of the `aws-efs-csi-driver-operator` deployment.
+
+. To update the tags in a hosted cluster, in the `HostedCluster` resource, enter information in the `resourceTags` fields, as shown in the following example:
++
+.Example `HostedCluster` resource
+[source,yaml]
+----
+apiVersion: hypershift.openshift.io/v1beta1
+kind: HostedCluster
+metadata:
+  #...
+spec:
+  autoscaling: {}
+  clusterID: <cluster_id>
+  configuration: {}
+  controllerAvailabilityPolicy: SingleReplica
+  dns:
+    #...
+  etcd:
+    #...
+  fips: false
+  infraID: <infra_id>
+  infrastructureAvailabilityPolicy: SingleReplica
+  issuerURL: https://<issuer_url>.s3.<region>.amazonaws.com
+  networking:
+    #...
+  olmCatalogPlacement: management
+  platform:
+    aws:
+      #...
+      resourceTags:
+      - key: kubernetes.io/cluster/<tag> #<1>
+        value: owned
+      rolesRef:
+        #...
+    type: AWS
+----
++
+<1> Specify the tag that you want to add to your Cluster Ingress Operator resource.
+
+* To enable tags to be updated for Amazon EBS volumes resources, complete the following steps:
+
+. In the `aws-efs-csi-driver-operator` service account, add two annotations, as shown in the following example. These annotations are required so that the {aws-short} EKS pod identity webhook that runs on the cluster can correctly assign {aws-short} roles to the pods that the Operator uses.
++
+[source,yaml]
+----
+eks.amazonaws.com/role-arn:<role_arn>
+eks.amazonaws.com/audience:sts.amazonaws.com
+----
+
+. Delete the operator pod or roll out a restart of the `aws-efs-csi-driver-operator` deployment.
+
+. To update the tags in a hosted cluster, in the `HostedCluster` resource, enter information in the `resourceTags` fields, as shown in the following example:
++
+.Example `HostedCluster` resource
+[source,yaml]
+----
+apiVersion: hypershift.openshift.io/v1beta1
+kind: HostedCluster
+metadata:
+  #...
+spec:
+  autoscaling: {}
+  clusterID: <cluster_id>
+  configuration: {}
+  controllerAvailabilityPolicy: SingleReplica
+  dns:
+    #...
+  etcd:
+    #...
+  fips: false
+  infraID: <infra_id>
+  infrastructureAvailabilityPolicy: SingleReplica
+  issuerURL: https://<issuer_url>.s3.<region>.amazonaws.com
+  networking:
+    #...
+  olmCatalogPlacement: management
+  platform:
+    aws:
+      #...
+      resourceTags:
+      - key: kubernetes.io/cluster/<tag> #<1>
+        value: owned
+      rolesRef:
+        #...
+    type: AWS
+----
++
+<1> Specify the tag that you want to add to your Amazon EBS volumes resource.
+
+* To enable tags to be updated for IAM user resources, complete the following steps:
+
+. In the `aws-efs-csi-driver-operator` service account, add two annotations, as shown in the following example. These annotations are required so that the {aws-short} EKS pod identity webhook that runs on the cluster can correctly assign {aws-short} roles to the pods that the Operator uses.
```

Comment:
@anirudhAgniRedhat what should I replace `aws-efs-csi-driver-operator` with for this part of the procedure?